src/modem/OwlModemNetwork.cpp

```diff
@@ -370,25 +370,6 @@ int OwlModemNetwork::setModemFunctionality(at_cfun_fun_e fun, at_cfun_rst_e *res
   return atModem_->doCommandBlocking(3 * 60 * 1000, nullptr) == AT_Result_Code__OK;
 }
 
-
```

Comment:
Getting/setting the MNO profile is required (setting to T-Mo in the US) as well as other options for other carriers/future roaming partners.  We should leave these in.

Reply:
I see what happened here, we are factoring it out to an RN4 specific class.  This will just need to be updated in OwlCLI to point to the new class until OwlCLI is extended to support modules other than the RN4.